src/models/methods/subscribe.ts

```diff
@@ -111,10 +111,25 @@ export interface ConsensusStream extends BaseStream {
   consensus: 'open' | 'establish' | 'accepted'
 }
 
+export interface PathFindStream extends BaseStream {
+  type: "path_find";
+  source_account: string;
+  destination_account: string;
+  destination_amount: Amount;
+  full_reply: boolean;
+  id: any;
```

Comment:
Streams don't have an ID - that's just the first response

Reply:
Unless that's special about `PathFind` - in which case it should be typed as `number | string`

Reply:
PathFind does have an ID